presto-native-execution/presto_cpp/main/tests/LinuxMemoryCheckerTest.cpp

```diff
@@ -33,10 +33,20 @@ class LinuxMemoryCheckerTest : public testing::Test {
 
   LinuxMemoryCheckerTest() : memChecker(PeriodicMemoryChecker::Config{}) {}
```

Comment:
Nit: not related to this PR but `memChecker` should probably be renamed to `memChecker_` as per the coding conventions.